src/pages/ThorChainLP/YourPositions.tsx

```diff
@@ -101,22 +137,32 @@ export const YourPositions = () => {
   const emptyIcon = useMemo(() => <PoolsIcon />, [])
   const isActive = true
 
+  const { parsedPools } = usePools()
+
   const positionRows = useMemo(() => {
     if (isActive) {
-      return (
-        <Stack mx={listMargin}>
-          <PositionButton />
-        </Stack>
-      )
+      // TODO(gomes): add a custom query hook for this and useQueries()
+      // this is a temporary way to achieve this
+      return parsedPools.map(pool => {
+        return (
+          <PositionButton
+            assetId={pool.assetId}
+            name={pool.name}
+            opportunityId={pool.opportunityId}
+            apy={pool.poolAPY}
```

Comment:
May or may not be the correct one to use https://gitlab.com/thorchain/midgard/-/merge_requests/564